node/manifest.ts

```diff
@@ -44,6 +44,17 @@ interface GenerateManifestOptions {
   userFunctionConfig?: Record<string, FunctionConfig>
 }
 
+const removeEmptyConfigValues = (functionConfig: EdgeFunctionConfig) =>
+  Object.entries(functionConfig).reduce((acc, [key, value]) => {
+    if (value && !(Array.isArray(value) && value.length === 0)) {
+      return { ...acc, [key]: value }
+    }
+    return acc
+  }, {} as EdgeFunctionConfig)
```

Comment:
```suggestion
  Object.fromEntries(Object.entries(functionConfig).filter(([_, value]) => value && !(Array.isArray(value) && value.length === 0)));
```

this is totally subjective, but I'm not a big fan of reduce, unless it is really reducing something. So I would find this much more readable and probably less objects created, but up to you.

Reply:
I think I would find this less readable because it hides a lot of logic and I need to read it a lot of times to understand what is happening. That's why I prefer the reduce option